R/metadata.R

```diff
@@ -351,21 +351,23 @@ orderly_shared_resource <- function(...) {
   invisible(files)
 }
 
-
 validate_shared_resource <- function(args, call) {
   if (length(args) == 0) {
     cli::cli_abort("'orderly_shared_resource' requires at least one argument",
                    call = call)
   }
-  assert_named(args, unique = TRUE, call = environment())
+
```

Comment:
Can you see if you can harmonise this through `validate_file_from_to` (in outpack_misc.R) which contains most of this logic?  I think it should just drop in directly?